service/collector/cluster.go

```diff
@@ -17,7 +17,7 @@ const (
 )
 
 var (
-	clusterDescription *prometheus.Desc = prometheus.NewDesc(
+	clusterStatus *prometheus.Desc = prometheus.NewDesc(
```

Comment:
I think the reason here was that `Desc` in Prometheus terms was thought to be some "description". I checked the godoc and it is in fact a "descriptor". So this here would be the cluster status descriptor.

Reply:
For me the following code reads nicer if `Description` is dropped out. I don't think it brings any added value in this case.

Reply:
All good. Just wanted to explain some background here.